cmd/rook/rook/rook.go

```diff
@@ -66,6 +68,7 @@ func init() {
 	RootCmd.PersistentFlags().StringVar(&logLevelRaw, "log-level", "INFO", "logging level for logging/tracing output (valid values: CRITICAL,ERROR,WARNING,NOTICE,INFO,DEBUG,TRACE)")
 	RootCmd.PersistentFlags().StringVar(&operatorImage, "operator-image", "", "Override the image url that the operator uses. The default is read from the operator pod.")
 	RootCmd.PersistentFlags().StringVar(&serviceAccountName, "service-account", "", "Override the service account that the operator uses. The default is read from the operator pod.")
+	RootCmd.PersistentFlags().StringSliceVar(&imagePullSecrets, "image-pull-secret", []string{}, "Override the image pull secrets section. Necessary to pull image from private repository.")
```

Comment:
Add an example of this usage to the various operator CRDs for the different backends.